examples/functions/trip_change/TripChange_001_Request_laterDeparture.xml

```diff
@@ -7,8 +7,13 @@
 			<OJPTripChangeRequest>
 				<siri:RequestTimestamp>2020-06-16T10:00:00Z</siri:RequestTimestamp>
 				<ChangeParams>
-					<ChangeLegRef>3</ChangeLegRef>
+					<ChangeLegRef>2</ChangeLegRef>
 					<EarlierArrivalLaterDeparture>laterDeparture</EarlierArrivalLaterDeparture>
+					<TripParams>
+						<NoStairs>true</NoStairs>
+						<NumberOfResults>3</NumberOfResults>
```

Comment:
NumberOfResults=3 does not make sense in my opinion, you are just changing one specified trip, so I would expect 1 trip as result.